### DISCUSSION TITLE
Should DataSaturday have a Speaker Diversity policy recommendation?

### DISCUSSION THREAD

**Discussion by SQLClause** [dataplat/DataSaturdays#307] [Ideas]
Kevin Chant created a pull request to add a Speaker Diversity policy to the repo - #305 
This needs to be a discussion first I believe - so adding the text here and opening up for a discussion.

# Speaker diversity policy

Due to the fact that we want our events to have a more diverse and inclusive range of speakers at events we recommend that you implement a speaker diversity policy and stick to it for your session selections. It is _optional_ but it will be appreciated in the community.

You can use the logo at {LOCATION (TBC)} on your promotions for your event if you do use a speaker diversity policy. Plus, to show this is _for_ the community _by_ the community we welcome other Microsoft Data Platform events to use this badge if they have a Speaker diversity policy and they stick to it.

Feel free to use the below as a template for your speaker diversity policy. Which is based on one created previously for another community event.

## Speaker diversity policy example
It is a simple fact that there are many speakers now willing to offer up their time and knowledge within the Microsoft Data Platform Community. This is amazing and something to be welcomed. 

However, a significant number of these speakers are male and white. In an effort to balance things out, {INSERT EVENT NAME HERE} will take necessary actions in the spirit of diversity. 

Examples of such actions are specific call-outs on social media, direct invitations to people, and other actions that we believe are appropriate. We also want to ensure that there is representation from less experienced speakers and we will take similar actions to encourage this. 

Put another way, we want to achieve a balance of experience and diversity across all of the talented speakers who participate at our event and we take steps to ensure that our session selections reflect this. 

This can mean, that some amazing technical speakers will not get a chance to speak, because that slot is filled by another amazing speaker in the name of diversity, opportunity, and promotion of work-life balance and mental health.

**Comment by tboggiano** [comment]
It does say it's optional, I personally would like more seen around developing diverse speakers.  I've blogged about some call for actions here: https://tracyboggiano.com/archive/2022/06/speaker-diversity/.  One way to do that is to do this.  I'd love to see more discussion around this as I didn't get any on my blog.
